tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_sfpu/ckernel_sfpu_log.h

```diff
@@ -61,6 +61,11 @@ sfpi_inline void calculate_log_body(const uint log_base_scale_factor) {
     }
     v_endif;
 
+    v_if (in < 0.0F) {
+        result = std::numeric_limits<float>::quiet_NaN();
+    }
+    v_endif;
+
```

Comment:
@ruthreshx Can you run the performance pipeline as well and make sure the extra conditional cycles have not had large effects on performance?

Reply:
Hi @rtawfik01 ,
May I know what all the workflows that needs to be run for performance check?

Reply:
This one for example: https://github.com/tenstorrent/tt-metal/actions/workflows/perf-device-models.yaml